internal/controller/status.go

```diff
@@ -2,11 +2,93 @@ package controller
 
 import (
 	"context"
+	"fmt"
+	"slices"
 
 	rsctv1alpha1 "github.com/ocp-power-automation/rsct-operator/api/v1alpha1"
 	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+type PodStatus string
+
+const (
+	PENDING           PodStatus = "PENDING"
+	RUNNING           PodStatus = "RUNNING"
+	FAILED            PodStatus = "RUNNING"
```

Comment:
why is this `"RUNNING"`? is this supposed to be `FAILED`?

Reply:
think its a typo, will correct it.